holidayextras/plugin-elephant
# NB-1332 Update to latest version of elephant

#### What does this PR do? (please provide any background)
[NB-1332](https://hxshortbreaks.atlassian.net/browse/NB-1332) Update elephant version to send events to redacted tables in BigQuery.

Also update lodash to fix vulnerability being reported by github.

See also https://github.com/holidayextras/the-works/pull/966

#### What tests does this PR have?
n/a

#### How can this be tested?
npm link in the-works
make a test booking
check the order in BigQuery - the schema version should say "v4.0.1"

#### Any tech debt?
no

#### Screenshots / Screencast

#### What gif best describes how you feel about this work?
![]()

- I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

By approving a review you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

---
